src/Common/src/Interop/Windows/Interop.UNICODE_STRING.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    [StructLayout(LayoutKind.Sequential)]
+    internal struct UNICODE_STRING
```

Comment:
I didn't reuse [this one](https://github.com/dotnet/corefx/blob/master/src/Common/src/Interop/Windows/sspicli/Interop.UNICODE_STRING.cs) because it marshals the buffer as string, and I am not sure that NtQuerySystemInformation always null terminates them.